app.js

```diff
@@ -70,6 +77,69 @@ function displayRepos(repos) {
     dinamicContent.appendChild(ul);
 }
 
+function displayIssues(issues, htmlElement) {
+    htmlElement.innerHTML = "";
+    let ul = document.createElement('ul');
+    let descendingIssuesByCreatedDate = [...issues];
+    descendingIssuesByCreatedDate.sort((a, b) => {
+        let timestampA = new Date(a.created_at).getTime();
+        let timestampB = new Date(b.created_at).getTime();
+
+        return timestampB - timestampA;
+    })
+
+    issues.forEach(issue => {
+        let li = document.createElement('li');
+        li.classList.add("list-none", "mt-4");
+
+        li.innerHTML = `
+        <div class="dialog-background"> 
+        <a href='${issue.html_url}' target='_blank' class="title-hover">
+            <span class="text-bold">
+            👉🏻 Title:
+            </span>
+            <span class="issue-title"></span>
+        </a>
+        <p> <span class="text-bold"> 🚀 Type: </span> ${issue.hasOwnProperty('pull_request') ? 'Pull Request' : "Issue"} </p>  
+        <p> <span class="text-bold"> 📅 Created at:</span> ${formatDateTime(new Date(issue.created_at))} </p>
+        <p> <span class="text-bold"> 🛠️ State:</span> ${issue.state} </p> 
+        ${issue.assignee !== null ? (`
+        <p> <span class="text-bold"> 👨🏻‍💻 Assignee:</span> ${issue.assignee.login} </p>`) : ""}
+        <p> <span class="text-bold"> 🕥 Last updated:</span> ${formatDateTime(new Date(issue.updated_at))} </p>
+        <p> <span class="text-bold"> ✅ Closed at:</span> ${formatDateTime(new Date(issue.closed_at))} </p> 
+        </div> 
+        `;
+
+        let titleElement = li.querySelector('.issue-title');
+        titleElement.innerText = issue.title;
+        ul.appendChild(li);
+    })
+    htmlElement.appendChild(ul);
+}
+
+function displayPaginationControls(hasPreviousPage, hasNextPage, onPrevious, onNext, pageNumber, htmlElement) {
+    let footer = document.createElement('footer');
+    footer.innerHTML = `
+    <div class="text-center"> 
```

Comment:
Why have this extra `div`. Wouldn't it be the same if we put the `text-center` class on the `footer` and then inside it directly the buttons and span?